### PR TITLE
epic_SciGlassCalorimeter -> epic_EcalBarrelSciGlass, BarrelSciGlassCa…

### PR DESCRIPTION
**Pull request by veprbl**
…lorimeter_geo.cpp -> BarrelCalorimeterSciGlass_geo.cpp

### Briefly, what does this PR introduce?

Rename to use same conventions as Interlayers

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
Yes. They might need to edit their private compact files

### Does this PR change default behavior?
No

### CODE REVIEW

File: src/BarrelCalorimeterSciGlass_geo.cpp [hunk 1]
```diff
@@ -410,4 +410,4 @@ static Ref_t create_detector(Detector &lcdd, xml_h handle,
   return det_element;
 }
 
-DECLARE_DETELEMENT(epic_SciGlassCalorimeter, create_detector)
+DECLARE_DETELEMENT(epic_EcalBarrelSciGlass, create_detector)
```
Comment (wdconinc) [inline src/BarrelCalorimeterSciGlass_geo.cpp:413]: We've been trying, to first order, to keep the name of the source file the same as the name of the element. Would we be able to use EcalBarrelSciGlass_geo.C?

Comment (veprbl) [reply src/BarrelCalorimeterSciGlass_geo.cpp:413]: Yes, I saw the inconsistency, but the naming convention for other files is not like that yet, so I opted to conform to that first.

Comment (veprbl) [reply src/BarrelCalorimeterSciGlass_geo.cpp:415]: Can you approve please.

Comment (wdconinc) [reply src/BarrelCalorimeterSciGlass_geo.cpp:415]: We should keep the name of the plugin source file consistent with the definition. Here you are moving from consistent to inconsistent names.

Comment (wdconinc) [reply src/BarrelCalorimeterSciGlass_geo.cpp:415]: Just to be clear, I like the rename of the plugin type and DECLARE_DETELEMENT, as it is clearer, but just would prefer the name of the src file to be `EcalBarrelSciGlass_geo.cpp` to be consistent with the others.

Comment (veprbl) [reply src/BarrelCalorimeterSciGlass_geo.cpp:415]: No, we shouldn't. The names of the files are not part of the visible API. Renaming all calorimeters is beyond the scope of this PR.

Comment (veprbl) [reply src/BarrelCalorimeterSciGlass_geo.cpp:415]: > Just to be clear, I like the rename of the plugin type and DECLARE_DETELEMENT, as it is clearer, but just would prefer the name of the src file to be `EcalBarrelSciGlass_geo.cpp` to be consistent with the others.

It's pretty consistent with others:
https://github.com/eic/epic/blob/15b2e4485ec8eafee6084889ca1d6eac20f47937/src/BarrelCalorimeterInterlayers_geo.cpp#L452

Comment (wdconinc) [reply src/BarrelCalorimeterSciGlass_geo.cpp:415]: Ok, let's make them consistent in a future PR then.

**Review by wdconinc — APPROVED** [review]

**Pull request merged**